### PR TITLE
Add option to pause

### PR DESCRIPTION
**Pull request by AmmNJ**
## Acceptance Criteria

- A button to pause the timer (share position with play button - new position for stop button)
- While paused, end time display is not visible - on restart, time display adapts to new calculation
- Pause button is not visible while countdown not previously started

### CODE REVIEW

File: src/GlobalStyle.js [hunk 1]
```diff
@@ -7,9 +7,24 @@ export default createGlobalStyle`
 
   body {
     margin: 0;
-    font-family: sans-serif;
+    font-family: 'Inter', sans-serif;
     line-height: 1.5;
     font-size: 112.5%;
     color: #585858;
```
Comment (melissaceleste) [inline src/GlobalStyle.js:13]: You could safe this in variables (if you haven't planned this allready for the next stories)

File: src/components/App/App.jsx [hunk 2]
```diff
@@ -11,6 +12,7 @@ function App() {
 
   const [durationLong, setDurationLong] = useState(false)
```
Comment (melissaceleste) [inline src/components/App/App.jsx:13]: why did you named it duration long? Why not just duration? Or is there also a duration short?

File: src/fonts/fonts.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { createGlobalStyle } from 'styled-components'
+
```
Comment (melissaceleste) [inline src/fonts/fonts.js:2]: I would delete the space

**Review by melissaceleste — COMMENTED** [review]
Hey, 
really nice and interesting code! God Job! 
I left just a few comments. Hope they are a little constructive :)

File: src/components/App/App.jsx (hunk 2 above)
Comment (AmmNJ) [reply src/components/App/App.jsx:13]: There is a short duration which is only applied, if there the state of the long duration is not active. The PR doesn't show the full code unfortunately.

**Pull request merged**